packages/blockchain-api/src/schema.ts

```diff
@@ -61,11 +61,14 @@ export interface CurrencyConversionArgs {
   sourceCurrencyCode?: string
   currencyCode: string
   timestamp?: number
+  knownExchangeRates?: { [pair: string]: string }
 }
 
 export interface MoneyAmount {
   value: BigNumber.Value
   currencyCode: string
+  // to override exchange rates from the APIs
+  knownExchangeRates?: { [pair: string]: string }
```

Comment:
Are multiple exchange rates needed? Also, could the rate be a number/BigNumber.value instead of a string, in order to be consistent with other types in this interface?